### ISSUE TITLE
Unable to get Spiders, Skeletons, Zombies or Enderman to spawn.

### ISSUE THREAD

**Issue by KitKatPanda** [DarkPacks/SkyFactory-4#1379]
## Bug Report
 No mobs other than Bouncy Creeper and Ultrons are spawning

## Expected Behaviour
 Expect to see all mobs spawn at night/in grinder

## Steps to Reproduce (for bugs)

1. Changed difficulty from hard to peaceful, then to easy, and normal, still nothing.
2. Restarted game and everything, still having the problem

## World Information

* Preset: Material Girl in a Material World

* Prestige: Yes
  
* Modpack Version world created in: 4.4.1.0 (1.12.2)

* Additional Content Installed:
Backpacks
Buildcraft (all)
Chickens
Hatchery
More Chickens
MrJTPCore
NEI
Optifine
Project E
Rebind Narrator
The One Probe
Vein Miner
World edit

## Client Information

* Modpack Version: 4.4.1.0 (1.12.2)
* Java Version: 8 Update 241
* Launcher Used: ATLauncher
* Memory Allocated: 10gb
* Server/LAN/Single Player: Singplayer
* Optifine Installed: Yes
* Shaders Enabled: No

**Comment by Searchcat37** [comment]
Just checking you were not near the grinder waiting for mobs to spawn.

**Comment by KitKatPanda** [comment]
> Just checking you were not near the grinder waiting for mobs to spawn.

No, I've got the grinder set about 10 blocks to the right of my base, and at least 25 blocks above. I should be far enough away for them to spawn, and indeed Bouncy Creepers and Ultrons spawn.. but that's it. No other mobs are spawning and I'm at a complete loss

**Comment by Rager10** [comment]
im on hard and no mobs are spawning im over 50 blocks away with no light on my base i even restarted minecraft launcher and CursedForge still didnt work.

**Comment by FrijDom** [comment]
I'm having the same issue, and from what I can find, the problem has to do with the biome being set as "The Void", and I assume it has to do with it not being a vanilla biome.

**Comment by TurtleModee** [comment]
Currently playing SF4 on a server with just me and a buddy and this bug is still occurring0.
